src/components/Auth/Signin.tsx

```diff
@@ -0,0 +1,112 @@
+import { zodResolver } from "@hookform/resolvers/zod"
+import { useForm } from "react-hook-form"
+import { z } from "zod"
+import { Eye, EyeOff } from 'lucide-react';
+import { Button } from "@/components/ui/button"
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form"
+import { Input } from "@/components/ui/input"
+import Auth from './AuthHeader'
+import ImageSlider from "./ImageSlider"
+import { useState } from "react"
+
+const UserSchema = z.object({
+  email: z
+    .string()
+    .email({ message: 'Invalid email address'}),
+  password: z
+    .string()
+    .min(6, {message: 'Passowrd is too short'})
+    .max(10, {message: "Password is too long"})
+})
+
+
+const Signin = () => {
+  const [showPassword, setShowPassword] = useState(false)
+  const form = useForm<z.infer<typeof UserSchema>>({
+    resolver: zodResolver(UserSchema),
+    defaultValues: {
+        email: '',
+        password: ''
+    }
+  })
+
+  const inputClass = 'rounded-none bg-slate-100 focus-visible:ring-0 focus-visible:ring-offset-0'
```

Comment:
1. ```bg-slate-100```  Due to setting background, on dark mode the text inside the input box is not visible.
2. Instead of writing the input classes into multiple component, can we write a custom class in some centralised place, like App.css

Reply:
Okay, i'll fix it.